docs/design-deployment.rst

```diff
@@ -25,6 +25,7 @@ The components that make up mite are:
 - recorder*: logs messages appearing on the mite message bus to a file
 - collector: listens for special messages on the bus and records them
```

Comment:
While you are here, can you swap collector and recorder?  This was a mistake from the original docs that I just noticed...